src/feature/Comments/Comments.tsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from 'react';
+import { useRequest } from '../../hooks/use-request';
+import * as Styled from './styles';
+
+import { Comment } from '../../types/types';
+import CommentCard from '../../components/CommentCard/CommentCard';
+
+type Props = {
+  reviewId: number;
+};
+
+type ResponseT = {
+  comments: Array<Comment>;
+};
+
+const Comments = ({ reviewId }: Props) => {
+  const [comments, setComments] = useState<Array<Comment>>();
+  const { sendRequest, isError, isLoading, errorMsg } = useRequest();
+
+  useEffect(() => {
+    const res = (data: ResponseT) => {
+      setComments(data.comments);
+    };
+
+    sendRequest('GET', `/reviews/${reviewId}/comments`, res);
+  }, []);
+
+  return (
+    <Styled.Comments direction="column">
+      {isError && <span>{errorMsg}</span>}
+
+      {isLoading && <span>Loading...</span>}
```

Comment:
Might just need to flip these around 
loading whilst api happen > either result or error then